src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
-            page.StandardOperators.ClearEntryButton.Click();
-            Assert.AreEqual("0", page.GetCalculatorResultText());
+            page.StandardOperators.NumberPad.Input(3);
+            page.StandardOperators.EqualButton.Click();
+
+            var historyPanelItems = page.HistoryPanel.GetAllHistoryListViewItems();
+            Assert.IsTrue(historyPanelItems[0].Text.Equals("2 Minus ( 3= Minus (1", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyPanelItems[1].Text.Equals("-3 + -2.6= Minus (5.6", StringComparison.InvariantCultureIgnoreCase));
+
+            Assert.AreEqual("-1", page.CalculatorResults.GetCalculatorResultText());
+
+            var historyItems = page.HistoryPanel.GetAllHistoryFlyoutListViewItems();
+            Assert.IsTrue(historyItems[0].Text.Equals("2 Minus ( 3= Minus (1", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyItems[1].Text.Equals("-3 + -2.6= Minus (5.6", StringComparison.InvariantCultureIgnoreCase));
+
+            Assert.AreEqual("-1", page.CalculatorResults.GetCalculatorResultText());
+
+            page.HistoryPanel.ListViewItem.SendKeys(Keys.ArrowDown + Keys.Enter);
+
+            Assert.AreEqual("-5.6", page.CalculatorResults.GetCalculatorResultText());
+            Assert.AreEqual("-3 + -2.6=", page.CalculatorResults.GetCalculatorExpressionText());
+
+            page.HistoryPanel.OpenHistoryFlyout();
+            page.HistoryPanel.ListViewItem.SendKeys(Keys.ArrowDown + Keys.ArrowUp + Keys.Enter);
+            Assert.AreEqual("-1", page.CalculatorResults.GetCalculatorResultText());
+
+            page.HistoryPanel.OpenHistoryFlyout();
+            page.HistoryPanel.ClearHistoryButton.Click();
+
+            page.HistoryPanel.OpenHistoryFlyout();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryEmpty"));
+
+            page.HistoryPanel.OpenHistoryPanel();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryEmpty"));
         }
+
         #endregion
 
-        #region Keyboard Input Tests
+        #region Error Tests
+        /// <summary>
+        /// These automated tests verify errors display gracefully
+        /// Via mouse input, all basic UI functionality is checked 
+        /// </summary>
+
         [TestMethod]
-        public void KeyboardInput_Subtract()
+        [Priority(1)]
+        public void Error_DivideByZero()
         {
-            page.Header.SendKeys("70090-890987=");
-            Assert.AreEqual("-820,897", page.GetCalculatorResultText());
+            page.StandardOperators.InvertButton.Click();
+            Assert.IsTrue(page.CalculatorResults.GetCalculatorResultText() == "Cannot divide by zero");
         }
 
         [TestMethod]
-        public void KeyboardInput_Multiply()
+        [Priority(1)]
+        public void Error_CubedNegativeNumber()
```

Comment:
Please update this test to match the name, should be ErrorSquareRootNegativeNumber

Reply:
Change will be in next commit.  Also, error testing was moved to CalculatorAppFunctionalTests.cs